src/main/resources/allowed-labels.properties

```diff
@@ -0,0 +1,71 @@
+# This file lists all the labels that are whitelisted
+
+administrative-monitor
```

Comment:
```suggestion
# Plugin governance labels. https://jenkins.io/doc/developer/plugin-governance/
adopt-a-plugin
adopt-me
deprecated

# Plugin categorization
administrative-monitor
agent
```

I suggest moving governance-related labels to the top. Also added `adopt-me` which looks to be quite popular among GitHub repos. Later we will need to deduplicate them, I'd guess (synonyms?)